src/pages/AddMembersPage/AddMembersPage.tsx

```diff
@@ -119,84 +139,67 @@ const AddMembersContents = ({
           Magic Link
         </TabButton>
         <TabButton
-          tab={Tab.ETH}
+          tab={Tab.CSV}
           currentTab={currentTab}
           setCurrentTab={setCurrentTab}
         >
-          ETH Address
+          CSV Import
         </TabButton>
-        {isFeatureEnabled('csv_import') && (
-          <TabButton
-            tab={Tab.CSV}
-            currentTab={currentTab}
-            setCurrentTab={setCurrentTab}
-          >
-            CSV Import
-          </TabButton>
-        )}
       </Flex>
       <Panel>
-        {currentTab === Tab.ETH && (
-          <Box>
-            <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
-              Add new members by wallet address.
-            </Text>
-
-            <NewMemberList
-              circleId={circle.id}
-              welcomeLink={welcomeLink}
-              revokeWelcome={revokeWelcome}
-            />
-          </Box>
-        )}
-        {currentTab === Tab.CSV && (
-          <Box>
-            <Box>CSV Import</Box>
-          </Box>
-        )}
-        {currentTab === Tab.LINK && (
-          <Box>
-            <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
-              Add new members by sharing a magic link.
-            </Text>
-            <ConstrainedBox css={{ mb: '$md' }}>
-              <Flex css={{ alignItems: 'center', mb: '$xs' }}>
-                <Text variant="label">
-                  Magic Circle Link
-                  {/*<ApeInfoTooltip>TODO: FILL THIS OUT</ApeInfoTooltip>*/}
+        <Box
+          css={{
```

Comment:
i think it looks great when there's no extra space in the right side of the box :)

Reply:
done